task/source-build/0.1/source-build.yaml

```
@@ -96,7 +96,7 @@ spec:
           --cachi2-artifacts-dir "$CACHI2_ARTIFACTS_DIR" \
           --registry-allowlist="$registry_allowlist"
 
-        cat "$RESULT_FILE" | jq -r ".image_url" >"$RESULT_SOURCE_IMAGE_URL"
-        cat "$RESULT_FILE" | jq -r ".image_digest" >"$RESULT_SOURCE_IMAGE_DIGEST"
+        cat "$RESULT_FILE" | jq -r ".image_url" | tr -d '\n' >"$RESULT_SOURCE_IMAGE_URL"
+        cat "$RESULT_FILE" | jq -r ".image_digest" | tr -d '\n' >"$RESULT_SOURCE_IMAGE_DIGEST"
```

Might be simpler to use `-j`:

>   -j, --join-output         implies -r and output without newline after
>                            each output;
 
```suggestion
        jq -j ".image_url" "$RESULT_FILE" >"$RESULT_SOURCE_IMAGE_URL"
        jq -j ".image_digest" "$RESULT_FILE" >"$RESULT_SOURCE_IMAGE_DIGEST"
```

---

Thanks. Changed it now.